chemked/chemked_schema.yaml

```diff
@@ -1,8 +1,9 @@
 # Common reference for authors' information
-author:
+author: &author
```

Comment:
Let's try your solution for `value-unit` with `author`, but wait until this PR is done (maybe do in your PR #7.